.github/workflows/concrete_python_checks.yml

```diff
@@ -10,8 +10,4 @@ jobs:
       - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
       - name: Pre-Commit Checks
         run: |
-          sudo apt install -y graphviz libgraphviz-dev
-          cd frontends/concrete-python
-          make venv
-          source .venv/bin/activate
-          make pcc
+          ./frontends/concrete-python/scripts/checks/checks.sh
```

Comment:
I made a script, I am never sure what happens if one of the commands fail

Reply:
Can you try it maybe by creating an intentional issue?

Reply:
I have seen CI where one of the step was RED but the flow continued. I think it's better to stop at the first RED and I'm afraid only the last status was taken into account (this latter thing might be wrong)